README.md

```diff
@@ -88,6 +88,9 @@ As of 2019, Pillow development is
             <a href="https://twitter.com/PythonPillow"><img
                 alt="Follow on https://twitter.com/PythonPillow"
                 src="https://img.shields.io/badge/tweet-on%20Twitter-00aced.svg"></a>
+            <a href="https://fosstodon.org/@pillow"><img
+                alt="Follow on https://fosstodon.org/@pillow"
+                src="https://img.shields.io/badge/publish-on%20Fosstodon-595aff"></a>
```

Comment:
Let's use the more general name Mastodon, and add in a `rel="me"` like in https://github.com/python-pillow/python-pillow.github.io/pull/29:

```suggestion
            <a href="https://fosstodon.org/@pillow"><img
                alt="Follow on https://fosstodon.org/@pillow"
                src="https://img.shields.io/badge/publish-on%20Mastodon-595aff"
                rel="me"></a>
```

I don't think this `rel="me"` will have any affect on the verified links, but just in case.

![image](https://user-images.githubusercontent.com/1324225/210602539-7adcaaf3-813c-48ff-9621-8159003ce6ec.png)